plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -92,17 +92,14 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         switch (connectorBehavior) {
             case SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY:
             case SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY:
-                return false;
-
             case SUPPORTS_TOPN_PUSHDOWN:
+            case SUPPORTS_JOIN_PUSHDOWN_WITH_DISTINCT_FROM:
+            case SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS:
```

Comment:
same here, the three options are not related

Reply:
Reverted